cli/outdated/outdated.go

```diff
@@ -59,10 +59,10 @@ func runOutdatedCommand(cmd *cobra.Command, args []string) {
 
 	// Prints outdated cores
 	tab := table.New()
-	tab.SetHeader(tr("Core name"), tr("Installed version"), tr("New version"))
+	tab.SetHeader(tr("Core name"), tr("Core ID"), tr("Installed version"), tr("New version"))
```

Comment:
On the other commands is just `ID`:

```
$ arduino-cli core search mkr1000
ID           Version Name                                        
arduino:samd 1.8.11  Arduino SAMD Boards (32-bits ARM Cortex-M0+)
$ arduino-cli core list
ID                       Installed Latest Name                                                                                 
adafruit:avr             1.4.13    1.4.13 Adafruit AVR Boards                                                                  
adafruit:nrf52           0.22.1    0.22.1 Adafruit nRF52                                                                       
adafruit:samd            1.7.2     1.7.2  Adafruit SAMD Boards                                                                 
arduboy-homemade:avr     1.6.23    1.6.23 Arduboy versions                                                                     
arduino-dev:sam          1.6.12    1.6.12 Arduino ARM (32-bits) Boards                                                         
arduino:avr              1.8.3     1.8.3  Arduino AVR Boards                                                                   
arduino-dev:mbed         1.3.0     1.3.0  Arduino Mbed OS Boards (nRF52840 / STM32H747)                                        
arduino:mbed_rp2040      2.1.0     2.1.0  Arduino Mbed OS RP2040 Boards                                                        
```

I'd keep the same label

Reply:
Also `Name` is in the latest position... we also have ID, versions, Name.

Maybe we can make the table layout the same?

Reply:
I used `Core ID` instead of only `ID` since it was already specified for the name too but also because this command also shows outdated libraries.
It's fine for me if we want to put the name at the end, would like to hear @ubidefeo opinion too.

Reply:
the column should be `ID` and be the leftmost